frontend/src/app/Routes.js

```diff
@@ -13,18 +13,20 @@ import Verify from '../views/Verify';
 
 function Routes() {
   return (
-    <Switch>
-      <Route path="/stores/new" component={NewStore} />
-      <Route exact path="/stores/:id/company" component={CompanyPage} />
-      <Route path="/stores/:id/product/:id?" component={ProductCreateEdit} />
-      <Route path="/stores/:id/onboarding/" component={Onboarding} />
-      <Route path="/stores" component={Stores} />
-      <Route path="/company/photo-select" component={PhotoSelect} />
-      <Route path="/order/:step" component={OrderForm} />
-      <Route path="/verify/:id/:token" component={Verify} />
-      <Route exact path="/" component={Home} />
-      <Route component={NoMatch} />
-    </Switch>
+    <BrowserRouter>
+      <Switch>
+        <Route path="/stores/new" component={NewStore} />
+        <Route exact path="/stores/:id/company" component={CompanyPage} />
+        <Route path="/stores/:id/product/:id?" component={ProductCreateEdit} />
+        <Route path="/stores/:id/onboarding/" component={Onboarding} />
+        <Route path="/stores" component={Stores} />
+        <Route path="/company/photo-select" component={PhotoSelect} />
+        <Route path="/order/:step" component={OrderForm} />
+        <Route path="/verify/:id/:token" component={Verify} />
+        <Route exact path="/" component={Home} />
+        <Route component={NoMatch} />
+      </Switch>
+    </BrowserRouter>
```

Comment:
I almost forgot Please remove this parent wrapper `BrowserRouter` because it's a duplication of what we have in the `index file`. The complete wrapper and the entry file  path is at `frontend/src/index.js`

Reply:
Resolved! Thanks for the feedback!